### PR TITLE
Bump bpmn js 8.3.1

### PR DESCRIPTION
**Pull request by MaxTru**
Related to https://github.com/bpmn-io/bpmn-js-properties-panel/pull/467

### CODE REVIEW

**Comment by nikku** [comment]
The thing we've just talked about, right? Glad to see it is an instance of a broken test case.

**Review by nikku — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -64,6 +64,6 @@
     "semver": "^6.3.0"
   },
   "peerDependencies": {
-    "bpmn-js": "^3.x || ^4.x || ^5.x || ^6.x || ^7.x || ^8.x"
```
Comment (nikku) [inline package.json:67]: Why this change? What actually broke in terms of compatibility?

**Review by nikku — CHANGES_REQUESTED** [review]
Too quick on the review. One question.

File: package.json (hunk 1 above)
Comment (MaxTru) [reply package.json]: Corrected

**Comment by nikku** [comment]
:clap:
